src/components/entry/icons/EatTheWeeds.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="31" height="20" viewBox="0 0 31 20">
+  <image width="31" height="20" xlink:href="data:img/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB8AAAAUCAYAAAB1aeb6AAAGfElEQVRIiVWW248cRxnFf3Xp7ume23p3Hce7TmyvbXzDDlHiizCGkASBFKS8BPHAG/BKJN6Bf4EHxFOkBBBCMhJIuUg4CiFCJDbEuZjYWV/wdXe9tnd3dmdmd6Zn+lKFqmYB0aOeanVX1Xfq1PnOV+Lo96bKoKKFNVJMHwwoC0WWalYX1xn0cqpjEclYTJGVxLWIpBERxSFSCKIkpL+eUhaGjfaQbivl6Fd2UQwL8jwjywzth12sENTGEhpbEt9v/toK0zMNpFTCzSNMaQhjjQ4EQgh0JDHGYo0FdwOWUesvIbDuU2mw1vpn1zogQoItSrI0w1iLcN2l8K2bxf9biy5zi4rA6oJOewBWMBxkrLd7WGGwBMhQkmV9lIX+0FDKDC01OqqhAomwUJqCdJD6FYc6IM9z8jKjJKfMBcImWFv4eQbDPmUZI77w0rg99uQxfvydn9Lur1CrNDHGYAto1Ju8+vYvWBus8sNvvMzyxn1Km9OsTrCWLmKV4ZPbH7K63uLZ/d8iosGl9nmuL13m8epOnjnwbWrhOK//4wyf3vs7Jw+e5Impp9kazvCnK2fQqiKoBDFP7TtFf9glUCGBrvyX3YO7DrOyvsypQ89TmpyizIiCKndan7HSX+DO0k2GWcqhnUeYGT/Cwwu3+fz+RYIgYMf4Yxya+jIf3TzP+TvvsTXeznOHXmSsOsW5ubfRWkmG2YAsH6BVxPXFy7x/+V2SsMp4c5KF1bt0N9b5zZ9/SVyNOXXgayytL3B16Z/cXLnC4tIDGmGTrMjppKusZx2vGQfQ7Xd/uE5/2MfFGZR9+lmfSjCiXheZAWkwtkTLgLnWTf547rckYY1mrUkv32DuwQJnL7zF8UMneeHJl1BScXb2D3x85wLDtODpx05QC+t+j3tZz4uxmWxlsjpNVgwojfGC6w97WGuQQiK0QodJQG5ySlOSFRnPf/FFvn74BU95Jazy8zd/xq3FXzHWGPPbMyj63OvcoKobfotUFDLXuuvHNuOEbfVHmb1/kWbcpDNYplYZY++jB3jnmmF6/HEa8Thp1mN++S5Sa4lx6YEkCesMiwHGlCihPIB+2kPJUZK4ZEnzLqu9Bz6ljMEr/WH7IdcezAKGg48cZefEHqqVmIquE6mEE/tOs2dyPydnnqOfdznzt9e4tzaPdnkX6NBTqbXmLxff4Hd/fYVapUGoKtxbnSeJq6RpShIlxEHN02YxFFlBpEOf1x9cf48TM88wUd3Gd7/0A/++mUx6b5ioPsLL3/wJM5OHmVu7wq/ffI3JPQkyzwqq1dgHd9fWxhRb6hMkUZVG0mTf9gNOlT5YpGMPtBJUEcIgnW0IgUJxY+FffDh7jh1b9jLd3E0c1pi9d5FLCxcIVMDuyf3eYOZac3T6HaQzGec0YRChdUhZGo7tOc3xvV9FycCD+fTWeT65830XA61Cv5K1dJmJZMrrJLAWqSR5lvPqG69w8sgpYlWn1V/g3c/eQlrJU7tP0+20qMVjvPP560gpCSoVtHtAQi/tUJjSoxNOjQjqcYMbi7Ok/dSbTndjjaXuAvVojK4Yerr/Y7NhEHJzZZ6P597nie3HmVu5xUe3P0BkitsPr1KJQs5e+j23lq4RyIgyLxGnfrTLhlGF8S2T2LwkjCN0RWMKS6hDWmvLrG10yIeWei1mesc2olrE6vIquuqAS3rd1AOev9ri2LMHSHRMq92i1Vpl0Bsy1thCLanRz9sYa7g92+axPTW0jgLvx0ud+5jSQk8RJ4psWHrzF0IhhcKanMxkrKyvUA+qpHlKnaovPp69zZozLAeUDH2WOAEH2tDutunnPeJIo6UrNBahNNqVPxUoKAVRpN1rgiBE2MLP51KqKEpG2pK4n7CCqDLSiFRqE+SoGiqhfWoKD2g0TvtMClDamYtLW1cxBb7QCSU8AGeHroQ6cTlHMi6ocyQ5mtile5aVvoy6d9YlOqOyqrQajTHWFyYxkoP3ATywzdYDhjLLkEVpRhNrSVzRVGLlFe06OyG5IE7Nrk8xNIxmHd15Vm6yvVmpjfHBR5XbjoBs1nnhycYv1C3CMebw+9WWufNcg3JQXQyvYuF14OkS+BW5GcrMoJSkcIoFBhuZjz+i//+PHSNYjkm8OckgRIeKIIndSUaOTiHGMBwahoMREw6Q2zY33O27ZyHwL/wWue/Omv02KRcZgtCx5kDaEd3+xPK/1sVwKVaWljwd8G8Z8jybYVC5EwAAAABJRU5ErkJggg=="/>
```

Comment:
this is massive

Reply:
Yes, this should be the original image, not converted to an SVG. But we can do this in a later PR. Please add a TODO somewhere in resources.js!

Reply:
^ @laurenho025